packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -459,11 +459,12 @@ Based on the scopes requested. Generally speaking, the scopes specified in a req
-| invalid_client           | The specified `client_id` isn't found.                                                                                                                                                                      |
-| invalid_grant            | The `code`, `refresh_token`, or `username` and `password` combination is invalid, or the `redirect_uri` doesn't match the one used in the authorization request.                                          |
-| invalid_request          | The request structure is invalid. For example, the basic authentication header is malformed, both header and form parameters are used for authentication, no authentication information is provided, or the request contains duplicate parameters. |
-| invalid_scope            | The scopes list contains an invalid or unsupported value.                                                                                                                                                  |
-| unsupported_grant_type   | The `grant_type` isn't `authorization_code`, `refresh_token`, or `password`.                                                                                                                               |
+| invalid_client                | The specified `client_id` isn't found.                                                                                                                                                                      |
+| invalid_grant                 | The `code`, `refresh_token`, or `username` and `password` combination is invalid, or the `redirect_uri` doesn't match the one used in the authorization request.                                          |
+| invalid_request               | The request structure is invalid. For example, the basic authentication header is malformed, both header and form parameters are used for authentication, no authentication information is provided, or the request contains duplicate parameters. |
+| invalid_scope                 | The scopes list contains an invalid or unsupported value.                                                                                                                                                  |
+| unsupported_grant_type        | The `grant_type` isn't `authorization_code`, `refresh_token`, or `password`.                                                                                                                               |
+| oie_not_enabled_for_interact  | The requested feature (Interaction Code flow) isn't enabled in this environment. Ensure that you have Okta Identity Engine enabled and the Interaction Code flow enabled for this application and authorization server.                                                                                                                               |
```

Comment:
I don't think we should add a new external error code for this. only internal one. Therefore I don't think we should document it.

Reply:
Alright, closing this PR with no change to dev-docs.